src/Navigation.js

```diff
@@ -6,83 +6,83 @@ class Navigation extends Component {
     return (
       <div>
         <Navbar bg="light" expand="lg">
-            <Navbar.Brand href="/">Home</Navbar.Brand> 
-            <Navbar.Toggle aria-controls="basic-navbar-nav" />
-            <Navbar.Collapse id="basic-navbar-nav">
-            <SignInUpMenu/>
+          <Navbar.Brand href="/"><img src="https://png.pngtree.com/svg/20161008/24671b749c.png" width='40px' height='45px' />Bookbeuty</Navbar.Brand>
```

Comment:
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images